M2/Macaulay2/packages/Macaulay2Doc/functions/factor-doc.m2

```diff
@@ -35,19 +35,19 @@ document {
 	  exponent is an integer.",
 	  },
      EXAMPLE {
-	  "factor 124744878111332355674003415153753485211381849014286981744945",
-	  "y = (2^15-4)/(2^15-5)",
-      	  "x = factor y",
-      	  "value x"
-	  },
+         "factor 124744878111332355674003415153753485211381849014286981744945",
```

Comment:
This test fails when using configure & make. Specifically, it progressively uses more and more memory until the process is terminated with status code 137. No error, just says "Killed".

Reply:
Interestingly, it does NOT fail when building and testing with cmake :/
I wonder if there's a linking issue somewhere?

Reply:
Ah, it was resolved after forcing M2 to build flint instead of using the one on the system. (Note: the flint-2.5.3.tar.gz is missing online)